components/esp_lvgl_port/test_apps/simd/README.md

```diff
@@ -4,6 +4,16 @@ Test app accommodates two types of tests: [`functionality test`](#Functionality-
 
 Assembly source files could be found in the [`lvgl_port`](../../src/lvgl9/simd/) component. Header file with the assembly function prototypes is provided into the LVGL using Kconfig option `LV_DRAW_SW_ASM_CUSTOM_INCLUDE` and can be found in the [`lvgl_port/include`](../../include/esp_lvgl_port_lv_blend.h)
 
+## Benchmark results
+
+| Color format | Matrix size | ANSI C version | ASM version    |
```

Comment:
I would also recommend adding memory alignment column. Because now it is a huge difference between `128x128` and `127x127` matrix just after changing matrix size.